lib/content-services/src/lib/node-comments/services/node-comments.service.ts

```diff
@@ -73,15 +75,25 @@ export class NodeCommentsService implements CommentsService {
     }
 
     private newCommentModel(comment: Comment): CommentModel {
+        const user = new User(comment.createdBy);
+        const userId = user.id?.toString();
+
+        const imageId = user.pictureId?.toString() || user.avatarId;
+
+        if (userId && imageId && !this.avatarCache.has(userId)) {
+            const avatarUrl = this.contentService.getContentUrl(imageId);
```

Comment:
Are you sure this endpoint will work with both `pictureId` and `avatarId`? I tested that for `hruser` and the request failed leaving the broken image